DevEdu.API/DevEdu.Business.Tests/RatingServiceTests.cs

```diff
@@ -392,12 +560,16 @@ public void UpdateStudentRating_StudentRatingDoesntExist_EntityNotFoundException
             var studentRatingId = 0;
             var value = 0;
             var periodNumber = 0;
-            var authorUserId = 0;
+            var authorUserInfo = new UserIdentityInfo

```

Comment:
please make a method that will return such objects
userId and userRole will be the parameters of this method